code/modules/mob/living/damage_procs.dm

```diff
@@ -224,6 +232,14 @@
 		updatehealth()
 	return amount
 
+/mob/living/proc/setFireLoss(amount, updating_health = TRUE, forced = FALSE)
+	if(!forced && status_flags & GODMODE)
```

Comment:
missing parenthesis. 

sidenote: man we should just make godmode checking a macro already...